setup.py

```diff
@@ -56,7 +56,7 @@ def get_requirements(filename):
                              'https://www.reddit.com/r/pics/',
                              'https://api.desktoppr.co/1/wallpapers', ]}
     with open(settings_file_path, "w") as settings_file:
-        settings_file.write(json.dumps(file_data))
+        settings_file.write(json.dumps(file_data) + '\n')
```

Comment:
hey so can you look at `update_settings` in `utils` ? i think that has the right way this just adds a a newline at then end of the json